content/docs/topics/charts.md

```diff
@@ -116,7 +116,7 @@ to `v1` and are still installable by Helm 3.
 
 Changes from `v1` to `v2`:
 
-- A `dependencies` field defining chart dependencies, which were located in a
+ - A `dependencies` field defining chart dependencies, which were located in a
```

Comment:
Can you please  remove the extra space before the `-`

Reply:
Sure, fixed it. A leftover from the previous rebase onto master.